plugins/notion/src/App.tsx

```diff
@@ -20,12 +20,18 @@ export function AuthenticatedApp({ context }: AppProps) {
 
     useEffect(() => {
         framer.showUI({
-            width: databaseConfig ? 340 : 325,
+            width: databaseConfig ? 360 : 325,
             height: databaseConfig ? 425 : 370,
+            // Only allow resizing when mapping fields as the default size could not be enough.
+            // This will keep the given dimensions in the Splash Screen.
+            resizable: Boolean(databaseConfig),
         })
     }, [databaseConfig])
 
     const synchronizeMutation = useSynchronizeDatabaseMutation(databaseConfig, {
+        onError(error) {
+            framer.notify(error.message, { variant: "error" })
```

Comment:
Instead of displaying an error in the field mapping UI we display a toast.

This seems way better because the error text was getting cut off. Sheets Plugin does the same. Probably we built this before we had such an API 😄